js/base/main.js

```diff
@@ -251,7 +251,37 @@ class RipeCommonsMainPlugin extends RipeCommonsPlugin {
             this.store.commit("format", this.ripe.format);
             this.store.commit("resolution", this.ripe.size);
             this.store.commit("backgroundColor", this.ripe.backgroundColor);
-            this.store.commit("ripeOptions", this.ripe.options);
+            this.store.commit("ripeOptions", {
+                options: this.ripe.options,

```

Comment:
@gcandal let's keep the name `ripeOptions` for the options and the other values let's name them `ripeState` instead